AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx

```diff
@@ -48,8 +58,18 @@ export default class SearchResult extends React.Component {
     getResultsHeader(results) {
         return results.length > 0
             ? (results.length === 1
-                ? <h3>{`${results.length} result`}</h3>
-                : <h3>{`${results.length} results`}</h3>)
+                ? <div className="result-heading"><h3>{`${results.length} result`}</h3><button onClick={this.jsPdfGenerator}>Download PDF</button></div>
+                : <div className="result-heading"><h3>{`${results.length} result`}</h3><button onClick={this.jsPdfGenerator}>Download PDF</button></div>)
```

Comment:
Previously the string was 'results' is there a reason for this change?

Reply:
Yes, this change adds the 'Download PDF' button to the search results beside the h3 heading.